libs/services/auth/testing/src/fixtures/fixture-factory.ts

```diff
@@ -600,6 +600,7 @@ export class FixtureFactory {
           providerId: delegationProvider.id,
           clients: [],
           provider: delegationProvider,
+          apiScopes: [],
```

Comment:
After you make this field CreationOptional in the `deletaion-type.model.ts` file you should be able to remove this.